### DISCUSSION TITLE
SG PINECONE FOREST 1604 Car Protocol?

### DISCUSSION THREAD

**Discussion by aseitz79** [pascallanger/DIY-Multiprotocol-TX-Module#920] [General]
Hi there,

does anybody know if the MPM already contains the protocol to drive the famous SG 1604 models from Pinecone?

I think with the new Radiomaster MT12 (4 in 1 version) a lot of request for suface models support might come up :-)

Thanks in advance,

Alex

**Comment by pascallanger** [comment]
You need to test all the protocols and sub protocols to find out if it's already implemented. If you look at the RF chip in the TX you might be able to reduce the research.

**Comment by pascallanger** [comment]
If you can't find a match then I need the TX + RX to be sent to my attention in France or a good participation for the purchase of a model.

**Comment by aseitz79** [comment]
![IMG20231219113029](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/52075616/3b891d6a-d007-4cc7-bfce-206ad803a7a5)

Thanks for the fast reply. I opened the TX and the only chip I found was the one in the attached image. Can you identify it? It reads:
PAN186CV
2035CTa

Kind regards,
Alex

**Comment by pascallanger** [comment]
PAN is usually indicating that it has a XN297 built in so definitively a protocol which could be reversed.
You can try all the protocols and subprotocols which have XN297 in the last column: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#available-protocol-table-of-contents-listed-alphabetically

**Reply by aseitz79** [reply]
Thanks, I tried every XN297 protocol without success :-(
Nevertheless, I could send you the RX & TX to reverse engineer. Would this be possible?

Kind regards,

Alex

**Reply by pascallanger** [reply]
You can contact me by email pascal _ langer @ yahoo . fr if you are still ok to ship me the TX and RX. I need a setup which I can play with out of the box, if that's just to power the RX and plug a servo it's fine but if that's an all in one I would need the full stuff.

**Reply by aseitz79** [reply]
@pascallanger Are there any news on the Pinecone protocol? Did you have the chance to reverse engineer the protocol?
